model/workflow_run_event_context_test.go

```diff
@@ -0,0 +1,69 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/google/go-github/v45/github"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWorkflowRunEventContext(t *testing.T) {
+	t.Run("Test PR", func(t *testing.T) {
+		context := newWorkflowRunEventContext(createWorkflowRunEvent(t, "workflow_run_event_pr.json"))
+		assert.Equal(t, "ab7a32c308ac42df77385bbb5e97f0e3aac5c42f", context.GetCommitHash())
+		assert.Equal(t, "", context.GetConclusion())
```

Comment:
I believe, a bit of redundant to test the empty string.

Reply:
For the queued workflow runs it is empty. Original Github event message.
![image](https://user-images.githubusercontent.com/98656635/186852981-2bd56110-45a8-494e-afcf-fffd804a1e00.png)

Reply:
I got it. I mean we can remove this line as we are not testing something specifically right? it's just an empty string